plenum/server/replica.py

```diff
@@ -2650,7 +2672,7 @@ def process_requested_prepare(self, prepare: Prepare, sender: str):
     def process_requested_commit(self, commit: Commit, sender: str):
         return self._process_requested_three_phase_msg(commit, sender, self.requested_commits)
 
-    def is_pre_prepare_time_correct(self, pp: PrePrepare) -> bool:
+    def is_pre_prepare_time_correct(self, pp: PrePrepare, sender: str) -> bool:
```

Comment:
It would be great to cover this by unit tests. See, for example, `test_replica_freshness.py`

Reply:
done